source/mdcii/mdcii/include/mdcii/gam/scene.hpp

```diff
@@ -21,6 +21,72 @@
 #include <inttypes.h>
 #include <string>
 
+#include "kontor.hpp"
+
+#define MAXSIZEKIND 5
+
+enum class ClimateType : uint8_t
+{
+  North = 0,
+  South = 1,
+  Random = 2,
+};
+
+enum class SizeType : uint8_t
+{
+  Size0 = 0,
+  Size1 = 1,
+  Size2 = 2,
+  Size3 = 3,
+  Size4 = 4
+};
+
+enum class NativeFlag : uint8_t
+{
+  NoNatives = 0,
+  Natives = 1
+};
+
+struct RandomIslands
+{
+  ClimateType climateType; // specifies the climate for "random" island generation
+  SizeType sizenr;         // specifies the size for "random" island generation
+  NativeFlag nativflg : 1; // unused?
+  uint8_t islandNumber;    // mixed increment with this number and the number field from ISLAND5 islands
+  uint16_t filenr;         // 0xFF means this island shall be choosen randomly between all present island files with the given size
+  uint16_t empty;          // empty
+  uint32_t posx;           // position
+  uint32_t posy;           // position
+};
+
+struct SceneSaveData
+{
+  uint8_t name[64]; // the name of the scenario
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [struct member 'SceneSaveData::name' is never used.](https://app.codacy.com/manual/armin.schlegel/mdcii-engine/pullRequest?prid=5498758)